cluster/addons/addon-horizontal-autoscaler/addon-horizontal-autoscaler.yaml

```diff
@@ -86,7 +86,7 @@ spec:
         kubernetes.io/os: linux
       containers:
       - name: autoscaler
-        image: registry.k8s.io/cpa/cluster-proportional-autoscaler:1.8.4
+        image: registry.k8s.io/cpa/cluster-proportional-autoscaler:v1.8.9
```

Comment:
We have just published a new image v1.9.0 that comes with the latest fix for scalability and CVEs.

https://github.com/kubernetes-sigs/cluster-proportional-autoscaler/releases/tag/v1.9.0

Reply:
Updated, thanks